### DISCUSSION TITLE
Environment Requirement

### DISCUSSION THREAD

**Discussion by cshmilyc** [NVlabs/curobo#77] [Software Q&A] [answered]
Hi~
I am trying to install and run curobo on my new computer.
So my first question is:
Does curobo has any constraints or requirement on the system platform?
Since the shell commands I saw seems to be run on Linux system,
So can I install curobo on windows? 
And is there any requirement or recommend for the release and kernel version for Linux?
How about the recommended version of python, pytorch and so on?
Thank you

**Comment by balakumar-s** [comment, marked as answer]
Some users have been able to run curobo on windows using WSL. The main platform we test on is Ubuntu 20.04/22.04 though any other linux platforms would also work. We require a NVIDIA GPU as cuRobo uses CUDA kernels to run computations for motion generation. 

Some package versions we have verified to work:
python 3.7 - 3.10 
pytorch 1.13+
